shipday/order/order_info.py

```diff
@@ -42,6 +42,7 @@ def __init__(self, *args, order_number: str = None, customer: Customer = None, p
         self._order_cost: OrderCost = order_cost or OrderCost(**kwargs)
         self._delivery_time: datetime = expected_delivery_time
         self._pickup_time: datetime = expected_pickup_time
+        self._delivery_instruction = kwargs['deliveryInstruction'] or kwargs['delivery_instruction']
```

Comment:
are we supporting both variant? any reason for that?

Reply:
yes vaia, camel case variant is specified in our api docs and snake case variant is what python developers are used to. Thats why added both versions for all these parameters

Reply:
yes both are supported, api docs specified camel case parameters for all, however, python devs are mostly used to snake case. Thats why I have added both variants for all the specified params.